docs/docs-components/Overview.js

```diff
@@ -1,24 +1,29 @@
 // @flow strict
 import { Fragment, useState, type Node } from 'react';
-import { Box, RadioGroup, Flex, Text } from 'gestalt';
+import { Box, Flex, SegmentedControl } from 'gestalt';
 import Page from './Page.js';
 import PageHeader from './PageHeader.js';
 import List from './OverviewList.js';
 import IllustrationContainer from './IllustrationContainer.js';
 import { type ListItemType } from './COMPONENT_DATA.js';
+import capitalizeFirstLetter from '../utils/capitalizeFirstLetter.js';
+
+const sortOrders = ['alphabetical', 'categorical'];
+
+type Props = {|
+  buildingBlockComponents?: $ReadOnlyArray<ListItemType>,
+  generalComponents: $ReadOnlyArray<ListItemType>,
+  platform: 'Web' | 'Android' | 'iOS',
+  utilityComponents?: $ReadOnlyArray<ListItemType>,
+|};
 
 export default function Overview({
   buildingBlockComponents,
   generalComponents,
   platform,
   utilityComponents,
-}: {|
-  buildingBlockComponents?: $ReadOnlyArray<ListItemType>,
-  generalComponents: $ReadOnlyArray<ListItemType>,
-  platform: 'Web' | 'Android' | 'iOS',
-  utilityComponents?: $ReadOnlyArray<ListItemType>,
-|}): Node {
-  const [order, setOrder] = useState('alphabetical');
+}: Props): Node {
+  const [order, setOrder] = useState<'alphabetical' | 'categorical'>('alphabetical');
```

Comment:
We should probably stay consistent b/w places we allow this sorting (like the Icon page) and either do Categorical or By category in both places

Reply:
Hm…"By Name" and "By Category"? I'm fine with whichever, I just don't like mixing forms with  "Alphabetical" and "Category"